source/aks/patching-artifactory.html.md.erb

```diff
@@ -45,28 +72,97 @@ If any issues arise during the update process, troubleshooting steps are provide
   kubectl describe pods -n artifactory artifactory-oss-0
   ```
 
+  • For further confirmation, ensure that the grep command below returns the chart version that you have applied.
+
+  ```command
+  kubectl describe pods -n artifactory artifactory-oss-0 | grep chart=artifactory-107.90.9
+  ```
+
+  • Ensure you can access [Artifacory](https://artifactory.sandbox.platform.hmcts.net/ui/repos/treeGeneral/hmcts) website. **Note:** you will need to be connected to F5 VPN to access this link.
+
 ### 3. Update CFT PTL
-  
-  • [Connect to PTL cluster](https://hmcts.github.io/cloud-native-platform/troubleshooting/index.html#connecting-to-aks-clusters)
 
-  • Remove 'artifactory-sbox.yaml' file and its corresponding code from kustomization.yaml.
+  • Remove `artifactory-sbox.yaml` file and its corresponding code from `kustomization.yaml` made in step 1.
 
-  • Update the version in 'artifactory.yaml' to latest version eg, (7.84.12).
+  • Update the version in `artifactory.yaml` to latest version eg, (7.90.9) <- ensure `10` is in front of the version.
 
-  ```command
-        containers:
-          - image: docker.bintray.io/jfrog/artifactory-oss:7.84.12
-            name: artifactory-oss
+  ```yaml
+  ...
+
+  chart:
+    spec:
+      chart: artifactory-oss
+      version: 107.90.9
+  
+  ...
   ```
   
-  • Raise a PR similar to the [Example](https://github.com/hmcts/cnp-flux-config/pull/32191/files) and get your PR approved and merged.
+  • Submit your PR and ensure it gets reviewed and merged.
   
-  • Repeat [step 2](#2-check-new-version-on-cft-ptlsbox) checking the new version on CFT PTL ensuring the pods are healthy.
+  • Repeat [step 2](#2-check-new-version-on-cft-ptlsbox) for **'CFT PTL cluster'** checking the Helm release is on the new version and ensuring the pods are healthy.
 
-### Related Links
+## Related Links
   
   [GitHub repo](https://github.com/hmcts/cnp-flux-config)
   
-  [Artifactory](https://artifactory.sandbox.platform.hmcts.net/ui/repos/tree/General/hmcts) **Note** you will need to be connected to F5 VPN to access this link.
+  [Artifactory](https://artifactory.sandbox.platform.hmcts.net/ui/repos/tree/General/hmcts) **Note:** you will need to be connected to F5 VPN to access this link.
 
   [Artifactory Release Information](https://jfrog.com/help/r/jfrog-release-information/artifactory-release-notes)
+
+## Troubleshooting
+
+### Missing required service
+
+```json
+[jfrou] [WARN] [217db729f7674ef0] [local_topology_helper.go:68] [main] [] - Missing required services: [jfmd]
+```
+
+You will see this error upon startup of artifactory as all services startup, if they do not clear after 60 - 90 seconds investigate further. This often occurs when one of the containers is not healthy, usually the router service. These errors most likely point to Master / Join key issues as the router cannot establish connection with the internal DB.
+
+### Join / Master key errors
```

Comment:
I would include here that often join/master key error is a false positive see -> https://github.com/jfrog/charts/issues/1917#issuecomment-2342990400

In this case, if it's worth looking into the bootstrapping process on the other containers to see the real error